juice/src/layers/common/linear.rs

```diff
@@ -215,10 +223,21 @@ impl<B: IBackend + LayerOps<f32>> ComputeParametersGradient<f32, B> for Linear {
             .unwrap();
 
         // gradient w.r.t bias
-        // Technically, the gradient of vector b of length n to itself is the I_n identity matrix,
-        // so instead we'll just copy the output_gradient[0] vector into
+        // The gradient of vector b of length n to itself is the I_n identity matrix,
+        // so multiply output_gradient[0] by a 1-column.
+        // Note that we instead multiply a one-row by output_gradient[0], since doing it
+        // in the opposite order causes gemm() implementation to miscalculate dimensions.
+        // TODO: Fix this.
```

Comment:
nit: move to an issue or do it right away :)

Reply:
Done.